src/domain/usecases/data-entry/ValidatePrimaryFileUseCase.tsx

```diff
@@ -27,8 +27,9 @@ export class ValidatePrimaryFileUseCase implements UseCase {
                     return this.egaspDataRepository.validate(inputFile, module.dataColumns);
                 });
             }
-            case "AMR - Individual": {
-                return this.risIndividualRepository.validate(inputFile);
+            case "AMR - Individual":
+            case "AMR - Funghi": {
+                return this.risIndividualFunghiRepository.validate(inputFile);
```

Comment:
Hi @davidbgomes, there is one extra column in AMR-I program that doesn't exist in AMR-Fungi program. So, there should be separate validations for each of these files.

- If AMR-I file has AB CLASS column missing, there should be a file validation error thrown here. 
- Similarly, if AMR-Fungi file has AB CLASS column present, there should be a file validation error thrown here.